flake.nix

```diff
@@ -45,6 +45,15 @@
         }).nixos.manual.x86_64-linux;
       };
 
+      # The "legacy" in `legacyPackages` doesn't imply that the packages exposed
+      # through this attribute are "legacy" packages. Instead, `legacyPackages`
+      # is used here as a substitute attribute name for `packages`. The problem
+      # with `packages` is that it makes operations like `nix flake show
+      # nixpkgs` unusably slow due to the sheer number of packages the Nix CLI
+      # needs to iterate through. But when the Nix CLI sees a `legacyPackages`
```

Comment:
We spoke a bit more on this this morning and the problem is more that, currently, Nixpkgs is a recursive attrset which is expensive to evaluate in its entirety, not (only) that `nix flake show` with thousands of packages is slow.

So essentially "iterat{e,ing} through" -> "evaluating" would mostly resolve my issues, I think.

Reply:
As expressed in that convo, ideally we'd have more thorough documentation that we can link to in lieu of this big inline comment, but I'm hoping that this is a reasonable interim solution